scripts/py_matter_idl/matter/idl/__init__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from . import backwards_compatibility as compatibility
+from . import generators, lint
+from . import matter_idl_types as types
+from .data_model_xml import ParseSource, ParseXmls
+from .generators.generator import FileSystemGeneratorStorage, GeneratorStorage
+from .matter_idl_parser import CreateParser
+
+__all__ = [
```

Comment:
Why did we flatten the exported parts? In particular:

- `backwards_compatibility` seems cleaner than `compatibility` (compatibility with what)
- IDL generally is the `matter files`. Pulling XMLs is a bit unclear as those are different types and should probably be separate namespace (especially for ParseSource ... what source)
- `types` seems a bit generic and likely to name clash

The others seem somewhat reasonable, however overally "why?"

Reply:
> The others seem somewhat reasonable, however overally "why?"

I've created this top level imports in order to simplify mater.idl imports in case of using this package directly from the source (without installing). Then the import can look like this:
```python
try:
    from matter import idl
except ImportError:
    sys.path.append(str(Path(__file__).resolve().parent / "py_matter_idl" / "matter"))
    import idl
```

Anyway, if it seems off for the idl package to have top level imports I will remove it. The try/except will be longer, though.

Reply:
After some consideration I've figured out that the best way of using matter-idl would be to simply install it. I was able to do it during bootstrap process. The matter-idl is installed in the editable mode, so all changes during development will be seen by command line scripts right away.